mteb/tasks/Audio/Clustering/eng/VoxPopuliAccentClustering.py

```diff
@@ -12,16 +12,14 @@ class VoxPopuliAccentClustering(AbsTaskAudioClustering):
         description="Clustering English speech samples by non-native accent from European Parliament recordings.",
         reference="https://huggingface.co/datasets/facebook/voxpopuli",
         dataset={
-            "path": "facebook/voxpopuli",
-            "name": "en_accented",
-            "revision": "719aaef8225945c0d80b277de6c79aa42ab053d5",
-            "trust_remote_code": True,
+            "path": "mteb/voxpopuli-accent-clustering",
```

Comment:
sidenote: voxpopuli also seems to have multiple languages (e.g., for language classification?)